mmv1/products/kms/terraform.yaml

```
@@ -98,6 +98,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       decoder: templates/terraform/decoders/kms.go.erb
       encoder: templates/terraform/encoders/kms_crypto_key.go.erb
       update_encoder: templates/terraform/update_encoder/kms_crypto_key.go.erb
+  CryptoKeyVersion: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+
+      Destroying a cryptoKeyVersion will not delete the resource from the project.
+    id_format: "{{crypto_key}}"
```

I think `id_format` can probably use `{{name}}`. If I recall correctly, this is the format used to determine the resource's id in the terraform state - using just `{{crypto_key}}` would result in id collisions - all versions under a particular crypto key would be considered the same resource.

```suggestion
    id_format: "{{name}}"
```

---

Will do, I'll confirm by running tests.